src/site/content/en/payments/android-payment-apps-delegation/index.md

```diff
@@ -1,36 +1,267 @@
 ---
 layout: post
-title: Support payment method change from an Android payment app
+title: Providing shipping and contact information from an Android payment app
 subhead: |
-  How to update your Android payment app to support payment method change with Web Payments APIs.
+  How to update your Android payment app to provide shipping address and payer's contact information with Web Payments APIs.
 authors:
   - sahel
 date: 2020-07-17
 updated: 2021-09-14
 description: |
-  Learn how to modify your Android payment app to provide the payment method change when the merchant has requested them via the Payment Request API.
+  Learn how to modify your Android payment app to provide the user's selected shipping address as well as contact information when the merchant has requested them via the Payment Request API.
 tags:
   - payments
 feedback:
   - api
 ---
 
-{% Aside 'warning' %}
+Entering shipping address and contact information through a web form can be a
+cumbersome experience for customers. It can cause errors and lower conversion
+rate.
 
-Shipping and address support in [the Payment Request API is removed from the
-specification](https://github.com/w3c/payment-request/pull/955) and is no longer
-functional in Android payment apps.
+That's why the Payment Request API supports a feature to request shipping
+address and contact information. This provides multiple benefits:
 
-{% endAside %}
+* Users can pick the right address with just a few taps.
+* The address is always returned in [the standardized
+  format](https://w3c.github.io/payment-request/#paymentaddress-interface).
+* Submitting an incorrect address is less likely.
+
+This functionality can be deferred to a payment app to offer a unified payment
+experience and it's called *delegation*.
+
+Whenever possible, Chrome delegates the collection of a customer's shipping
+address and contact information to the invoked Android payment app. The
+delegation reduces the friction during checkout because the user's installed
+payment apps usually have more accurate information about their shipping address
+and contact details.
```

Comment:
Do they? Is there any reason to assume that they "usually" have more accurate information, especially if people are more used to storing multiple shipping addresses in merchant websites?